### PR TITLE
Allow Tiqr second factors to be vetted

### PR DESCRIPTION
**Pull request by rjkip**
- [x] Translations
- [ ] https://github.com/SURFnet/Stepup-Gateway/pull/38

### CODE REVIEW

**Comment by rjkip** [comment]
> Chocolate pudding
> It no phase me
> But you got cookie
> So share it maybe

Ping @DRvanR

File: src/Surfnet/StepupRa/RaBundle/Service/GssfService.php [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+        if (!$verification) {
+            return VerificationResult::noSuchProcedure();
+        }
+
+        if ($gssfId === $verification['gssfId']) {
+            return VerificationResult::verificationSucceeded($verification['procedureId']);
+        }
+
+        return VerificationResult::verificationFailed($verification['procedureId']);
```
Comment (DRvanR) [inline src/Surfnet/StepupRa/RaBundle/Service/GssfService.php]: :heart_eyes:

File: src/Surfnet/StepupRa/SamlStepupProviderBundle/DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+                                    ->isRequired()
+                                    ->info('The absolute path to the public key used to sign AuthnRequests')
+                                ->end()
+                                ->scalarNode('private_key')
+                                    ->isRequired()
+                                    ->info('The absolute path to the private key used to sign AuthnRequests')
+                                ->end()
+                            ->end()
+                        ->end()
+                        ->arrayNode('identity_provider')
```
Comment (DRvanR) [inline src/Surfnet/StepupRa/SamlStepupProviderBundle/DependencyInjection/Configuration.php]: RA should not be able to be an IdP :wink:

File: src/Surfnet/StepupRa/SamlStepupProviderBundle/DependencyInjection/SurfnetStepupRaSamlStepupProviderExtension.php [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+            new Reference('surfnet_saml.signing_service'),
+            new Reference('gssp.provider.' . $provider . 'metadata.configuration')
+        ]);
+        $container->setDefinition('gssp.provider.' . $provider . '.metadata.factory', $metadataFactory);
+    }
+
+    private function createRouteConfig($provider, $routeName)
+    {
+        // In the future, we ought to wrap this in an object.
+        // https://www.pivotaltracker.com/story/show/90095392
```
Comment (DRvanR) [inline src/Surfnet/StepupRa/SamlStepupProviderBundle/DependencyInjection/SurfnetStepupRaSamlStepupProviderExtension.php]: muchos gracias

**Comment by DRvanR** [comment]
:+1:

**Pull request merged**